modules/test/playwright/tests/object-web/utils/mockObjectFieldsObjectEntry.ts

```diff
@@ -0,0 +1,362 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {getRandomInt} from '../../../utils/getRandomInt';
+import getRandomString from '../../../utils/getRandomString';
+
+type objectFieldBusinessType = (typeof objectFieldBusinessTypes)[number];
+
+type objectFieldBusinessTypesLabelName = {
+	[K in objectFieldBusinessType]: objectFieldLabelName;
+};
+
+type objectFieldLabelName = {
+	label: string;
+	name: string;
+};
```

Comment:
Hey @igor-franca, can you create a LabelNameObject type call in `object.d.ts`? This kind of type can be reusable in the future